rest-e2e-tests/products-api.js

```diff
@@ -12,6 +12,11 @@ const DEVOPS_USER_TOKEN = 'devops-user-key'
 const Streamr = initStreamrApi(URL, LOGGING_ENABLED)
 const schemaValidator = new SchemaValidator()
 
+function assertIsPermission(data) {
```

Comment:
% npm run test-rest-api

Fails with errors on my machine.

Reply:
all tests pass for me

Reply:
All tests pass for me too!